packages/gateway-client/src/worker/index.ts

```diff
@@ -764,10 +768,7 @@ self.addEventListener('fetch', function (event) {
 self.addEventListener('online', () => console.log('<<<<online'));
 self.addEventListener('offline', () => console.log('<<<<offline'));
 
-const messageHandlers: Record<
-    ClientMessageType,
-    (msg: Message<ClientMessageType>) => void
-> = {
+self.messageHandlers = {
```

Comment:
Why are we making the message handlers global?

Reply:
this is leftover from websocket tinkering, will fix.